### ISSUE TITLE
Tune Hyperparameters

### ISSUE THREAD

**Issue by godekdls** [godekdls/korean-thematic-analysis#26]
## In the case of under fitting:
Adding features
Adding polynominal features
Decreasing lambda
Increasing number of layers
Decreasing dropout rate
Decreasing learning rate

## In the case of over fitting:
Getting more training examples
Trying smaller sets of features (the num of min top K in ngram selector)
Increasing lambda
Decreasing number of layers
Increasing dropout rate

## ??
number of units per layer 8, 16, 32, 64...

## If the training is too slow
Increasing learning rate
